src/kv/change_set.h

```diff
@@ -4,18 +4,20 @@
 
 #include "ds/champ_map.h"
 #include "ds/hash.h"
+#include "ds/rb_map.h" // TODO: Remove include
 #include "kv/kv_types.h"
 
 #include <map>
 
 namespace kv
 {
+  // TODO: Revert to CHAMP for now
```

Comment:
To revert before merging. I just want to make sure CCF works fine when using the `RBMap`.